include/ck_tile/ops/gemm/pipeline/gemm_universal_pipeline_ag_bg_cr_policy.hpp

```diff
@@ -22,286 +22,128 @@ struct UniversalGemmPipelineAgBgCrPolicy
 
     static constexpr bool TransposeC = true;
 
+    template <typename Problem, typename DataType, index_t kMNPerBlock>
+    CK_TILE_HOST_DEVICE static constexpr auto GetKPack()
```

Comment:
This name is causing confusion. This function calculates vector load size. Off course you could use then this result for KPack. But this function is more general.